packages/sdk/src/stores/google-datastore/store.ts

```diff
@@ -4,38 +4,38 @@ import { Datastore, Key } from "@google-cloud/datastore";
 import highland from "highland";
 
 // TODO: this is a work in progress which will be completed during datastore integration.
-export default function <D>(kind: string, store: Datastore): SortedStore<string, D> {
-  function makeKey(id: string): Key {
+export default function <I extends string | number, D>(kind: string, store: Datastore): SortedStore<I, D> {
```

Comment:
i was afraid this might have to change, all good

Reply:
yeah, Datastore accepts string and number as key type